src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealSubmittedConfirmation.java

```diff
@@ -31,22 +34,39 @@ public PostSubmitCallbackResponse handle(
             throw new IllegalStateException("Cannot handle callback");
         }
 
+        final AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
         PostSubmitCallbackResponse postSubmitResponse =
             new PostSubmitCallbackResponse();
 
+        String payForAppealOfflineOption = null;
```

Comment:
May be worth renaming this field to `paymentOption`

Reply:
Renamed to paymentOption